### DISCUSSION TITLE
Provide syntax highlighting module

### DISCUSSION THREAD

**Discussion by mandric** [microsoft/monaco-editor#3916] [Q&A]
I am creating a blog and would like to use a comparable syntax highlighting theme used in VS Code.  

Could you create a separate project for the JS/TS modules that do syntax highlighting so it can be imported on a web page, similar to HighlightJS or PrismJS?

**Comment by drewdaemon** [comment]
We could use this, too. We [rolled a custom theme](https://github.com/elastic/kibana/pull/186345) for our new query language, ES|QL. Would love to have the same syntax highlighting in our docs.
